Ginger/GingerCoreNETUnitTest/GingerRunnerTests/GingerRunnerTest.cs

```diff
@@ -464,5 +466,59 @@ public RunSetConfig CreteRunsetWithOperations()
 
             return runSetConfig;
         }
+
+
+        [TestMethod]
+        [Timeout(60000)]
+        public void RunDisabledActivityTest()
+        {
+            //Arrange
+            Activity activity = GetActivityFromRepository();
+
+            //Act
+            mGR.RunActivity(activity);
+
+            //Assert
+            Assert.AreEqual(activity.Status, Amdocs.Ginger.CoreNET.Execution.eRunStatus.Skipped);
+           
+        }
+
+        [TestMethod]
+        [Timeout(60000)]
+        public void RunDisabledActionTest()
+        {
+            //Arrange
+            Activity activity = GetActivityFromRepository();
+
+            ObservableList<IAct> actionList = activity.Acts;
+            Act action = (Act)actionList[0];
+            action.Active = false;
+
+            //Act
+            mGR.RunAction(action);
+
+            //Assert
+            Assert.AreEqual(action.Status, Amdocs.Ginger.CoreNET.Execution.eRunStatus.Skipped);
+            Assert.AreEqual("Action is not active.", action.ExInfo);
+        }
+
+        public Activity GetActivityFromRepository()
+        {
+            context = new Context();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make the enclosing instance method 'static' or remove this set on the 'static' field.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4755016)